src/App.module.scss

```diff
@@ -1,4 +1,54 @@
-.smth {
-  color: aquamarine;
-  background-color: red;
+$customGreen: #2cc56c;
```

Comment:
create folder **styles**, which should include **_mixins.scss, _variables.scss**
create mixins and use mixins for reusable components
replace all color variables to a new file **_variables.scss**

Reply:
use camelCase for classNames, not underscore.